src/index.html

```diff
@@ -7,8 +7,38 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="hover">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo" href="index.html">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="navbar">
+          <li class="nav-list"><a class="links is-active" href="/">apple</a></li>
+          <li class="nav-list"><a class="links" href="/">samsung</a></li>
+          <li class="nav-list"><a class="links" href="/">smartphones</a></li>
```

Comment:
Would be better to do like that, easy to read or add something else
```suggestion
          <li class="nav-list">
            <a class="links is-active" href="/">apple</a>
          </li>
          
          <li class="nav-list">
            <a class="links" href="/">samsung</a>
          </li>
          
          <li class="nav-list">
            <a class="links" href="/">smartphones</a>
          </li>
```